librz/core/canalysis.c

```diff
@@ -6393,3 +6393,34 @@ RZ_API ut64 rz_core_prevop_addr_force(RzCore *core, ut64 start_addr, int numinst
 	}
 	return start_addr;
 }
+
+RZ_API void rz_core_perform_auto_analysis(RzCore *core, RzCoreAnalysisType type) {
```

Comment:
before it was not an `RZ_API` so there was no need for this but you need to add these changes:
```suggestion
RZ_API void rz_core_perform_auto_analysis(RZ_NONNULL RzCore *core, RzCoreAnalysisType type) {
	rz_return_if_fail(core);
```